typescript_test/typescript_extended/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "module": "commonjs",
     "strict": true,
+    "importsNotUsedAsValues": "error",
```

Comment:
This should not be needed since this `tsconfig` extends the one from the parent directory. Still, all the other compiler options are duplicated here so I added this one as well.